test/provisioner/provision_azure_initializer.go

```diff
@@ -56,6 +56,8 @@ func initAzureProperties(properties map[string]string) error {
 		Location:          properties["LOCATION"],
 		SshPrivateKey:     properties["SSH_KEY_ID"],
 		CloudProvider:     properties["CLOUD_PROVIDER"],
+		ImageID:           properties["AZURE_IMAGE_ID"],
+		SubnetID:          properties["AZURE_SUBNET_ID"],
```

Comment:
Can the Subnet ID be retrieved from the AKS cluster ?
Otherwise AZURE_SUBNET_ID needs to be added to the properties file.

Reply:
Subnet id is retrieved from AKS cluster. Not sure how this error is coming. Subnet ID to set to kustomization yaml via AzureProps.